### PR TITLE
UIIN-2514: Search box/Browse box- Reset all should shift focus back to search box 

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Search box/Browse box- Reset all should shift focus back to search box 

Pass refs to `<SearchAndSort>` and use them to focus search box after reset

## Screenshots

https://github.com/folio-org/ui-inventory/assets/19309423/f773b3f2-3103-48cf-90ea-daf70c266dfd

## Issues
[UIIN-2514](https://issues.folio.org/browse/UIIN-2514)

## Related PRs
https://github.com/folio-org/stripes-smart-components/pull/1413
https://github.com/folio-org/stripes-components/pull/2169
https://github.com/folio-org/stripes-acq-components/pull/727

### CODE REVIEW

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 * Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
 * Added support for `containsAny` match option in Advanced search. Refs UIIN-2486.
 * Inventory search/browse: Do not retain checkbox selections when toggling search segment. Refs UIIN-2477.
+* Show Instance record after creating with Fast add option. Refs UIIN-2497.
+* Search box/Browse box- Reset all should shift focus back to search box. Refs UIIN-2514.
```
Comment (Dmytro-Melnyshyn) [inline CHANGELOG.md:10]: Is the Browse box focused?

Comment (BogdanDenis) [reply CHANGELOG.md:10]: Added a PR for Browse
https://github.com/folio-org/stripes-acq-components/pull/727

**Comment by mariia-aloshyna** [comment]
Fix eslint comments and tests, please

**Comment by BogdanDenis** [comment]
@mariia-aloshyna fixed tests

**Pull request merged**